publish-wrapper

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python3
```

Comment:
Lets bikeshed about the name of this script :)

Reply:
blue :bike: :house: !

I thought of "sink-logger", but that doesn't capture the GitHub statuses aspect. publish-wrapper is good enough for now *shrug*